CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ v1.5.0-rc.0
 
 ### Features
 
+- Add support bundle generation via the API endpoint /-/support (@dehaansa)
```

Comment:
I wonder if we should leave a note somewhere in docs that support bundles are not covered by backwards compatibility guarantees?